runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java

```diff
@@ -250,7 +251,7 @@ private boolean processCheckpointBatchByEntry(List<Long> batch,
         } catch (TrimmedException ste) {
             // The valid checkpoint has been trimmed.
             if (options.ignoreTrimmed) {
-                log.warn("processCheckpoint[{}]: Ignoring trimmed exception for address[{}]," +
+                log.debug("processCheckpoint[{}]: Ignoring trimmed exception for address[{}]," +
```

Comment:
In the case of ignoreTrimmed is this misleading ?

Reply:
That's why we are changing the log level. As it only ignores it if it was explicitly set to be ignored. Shouldn't then be something to worry about or notify. If ignored trimmed flag is set,  this stream is never reset to load from a checkpoint which means that this message is likely to appear a lot of times.